src/snakefile

```diff
@@ -118,7 +120,7 @@ rule lpFile:
 ## INTERMEDIATE RULES
 #####################################
 
-rule ConfigFile:
+rule config:

```

Comment:
Lets change this to `configRule` (or `ConfigRule`, whatever convention you are using). When we build in the configuration file to the workflow, we import it using the `config` or `configFile` keyword I think. So just improves clarity (my original of name of `ConfigFile` for the rule probably wasnt the best option either)